### PR TITLE
feat: add favicon

### PR DESCRIPTION
**Pull request by djaiss**
Perhaps the most important PR of this entire repo.

### CODE REVIEW

File: resources/views/app.blade.php [hunk 1]
```diff
@@ -8,24 +8,28 @@
...
-        'tracesSampleRate' => config('sentry.traces_sample_rate'),
-      ]); !!}
+      const SentryConfig = {
+        !!\json_encode([
+          'dsn' => config('sentry.dsn'),
+          'environment' => config('sentry.environment'),
+          'sendDefaultPii' => config('sentry.send_default_pii'),
+          'tracesSampleRate' => config('sentry.traces_sample_rate'),
+        ]);!!
+      }
```
Comment (asbiin) [inline resources/views/app.blade.php]: This is not good, Laravel expect that:
```suggestion
      const SentryConfig = {!! \json_encode([
        'dsn' => config('sentry.dsn'),
        'environment' => config('sentry.environment'),
        'sendDefaultPii' => config('sentry.send_default_pii'),
        'tracesSampleRate' => config('sentry.traces_sample_rate'),
      ]); !!}
```

**Review by asbiin — CHANGES_REQUESTED** [review]
Not sure about the size of the favicon, isn't it supposed to be very light?

File: resources/views/app.blade.php (hunk 1 above)
Comment (djaiss) [reply resources/views/app.blade.php]: Well, VSCode somehow has changed that... I didn't touch it myself.

**Comment by djaiss** [comment]
The size is big because it's an svg. So it appears perfectly on 2x screens.

**Review by asbiin — APPROVED** [review]

**Pull request merged**